recipe/meta.yaml

```diff
@@ -11,19 +11,18 @@ source:
   sha256: 8b682ed08caabfb7c042d4be083e28ddc692afb1deff5567111f8855071b75c3
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
     - pip
     - python
-    - setuptools >=42.0
-    - setuptools-scm >=3.5.0
+    - setuptools
+    - setuptools-scm 6.0.1
```

Comment:
Missing `toml`, see https://codeberg.org/miurahr/pybcj/src/commit/40ea774934b828c79874d387383eac0b422d5fb8/pyproject.toml#L58

Reply:
`toml`/`tomli` is already installed unconditionally by our `setuptools_scm` recipe. I see no need to include it in each package and polute the dependency graphs with sub-dependencies.